Assets/NuGet/Editor/NugetConfigFile.cs

```diff
@@ -306,11 +334,166 @@ public static NugetConfigFile CreateDefaultFile(string filePath)
        <add key=""repositoryPath"" value=""./Packages"" />
        <add key=""DefaultPushSource"" value=""http://www.nuget.org/api/v2/"" />
     </config>
+";
+
+            // add ignored
+
+            contents +=
+@"    <ignoredDependencies>" + "\n";
+
+            foreach (var iPackageID in dependenciesUnityGet())
+            {
+                contents += string.Format(
+@"       <add value=""{0}"" />" + "\n", iPackageID
+                );
+            }
+
+            contents +=
+@"    </ignoredDependencies>
 </configuration>";
 
             File.WriteAllText(filePath, contents, new UTF8Encoding());
 
             return Load(filePath);
         }
+
+        /// <summary>
+        /// Get all the dependencies that are referenced in Unity assemblies by default;
+        /// </summary>
+        private static IEnumerable<string> dependenciesUnityGet()
+        {
+            return new HashSet<string>(){
```

Comment:
How was this list generated? I'm concerned that it could be difficult to maintain this data. Would it be possible for this function to dynamically generate this by reflecting on Unity's assemblies? If not, can we document a set of steps for updating the list?